tests/tests.js

```diff
@@ -23,5 +23,23 @@ describe("Starting tests for app.js ->", () => {
       });
   });
 
+  it("To check if /factorial endpoint is working", (done) => {
+    chai
+      .request(app)
+      .post("/factorial")
+      .type("form")
+      .send({"number": 9})
+      .end((err, res) => {
+        if (err) {
+          done(err);
+          process.exit(1);
+        } else {
+          res.should.have.status(200);
+          res.body.should.have.property('result');
```

Comment:
Add validation for the logic as well. For example factorial of 9 is 362880. So check if the response consists of this output. 
Hint: use res.body.should.equal

Reply:
Fixed conflict and added script for checking the output. Kindly check. Thanks!